lib/form_helpers.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+/// FormHelpers contains mutliple methods to avoid code repition.
+/// @author: Ameer Ghazal
+class FormHelpers {
+  /// Genric function that checks if the user has input any value into the respective box.
+  /// @return: returns a list of widgets to notify the user of requirment, if applicable.
+  /// @author: Ameer (AG)
+  static List<Widget> checkError(String field) {
```

Comment:
naming things is hard, but checkError doesn't sound like it returns an error string. 

I think I would use something like buildOptionalError